api/v1alpha1/backup_types.go

```diff
@@ -61,7 +82,7 @@ type BackupSpec struct {
 	// Compression algorithm to be used in the Backup.
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
I suggest we add an enum annotation here, to it maps to an OpenAPI enum. It would be something like:

```bash
	// +kubebuilder:validation:Enum=none;zlib;gzip
```

Reply:
See an example here:
https://github.com/mariadb-operator/mariadb-operator/blob/6171b061f944e10e29197fbdc7e4d1144b08352d/api/v1alpha1/mariadb_galera_types.go#L360

Reply:
done, thx for the tip